lib/generators/slim/scaffold/scaffold_generator.rb

```diff
@@ -2,21 +2,24 @@
 
 module Slim
   module Generators
-    class ScaffoldGenerator < Erb::Generators::ScaffoldGenerator
+    class ScaffoldGenerator < Erb::Generators::ScaffoldGenerator # :nodoc:
```

Comment:
Because it is best practise and standard to add class level documentations. However, on Rails repository they don't document templates and generator code.

PS: I also used string style like on Rails repository. You do not have RuboCop on this repository, but I do use it with Rails defaults.

Reply:
Thanks, I see, this gem only makes sense when used with Rails so I think we can add Rubocop with Rails rules. Let me review first.